src/main/kotlin/com/cognifide/gradle/aem/common/instance/service/pkg/PackageManager.kt

```diff
@@ -100,11 +103,12 @@ class PackageManager(sync: InstanceSync) : InstanceService(sync) {
 
     /**
      * Create package on the fly, upload it to instance then build it.
-     * Finally download built package by replacing it with initially created.
+     * Next built package is downloaded - replacing initially created package.
+     * Finally built package is deleted on instance (preventing messing up).
      */
     fun download(definition: PackageDefinition.() -> Unit, retry: Retry): File {
         val file = aem.composePackage {
-            version = "download" // prevents CRX package from task 'Compose' being replaced
+            version = Formats.dateFileName()
```

Comment:
clue of change. if we wanted to create packages in loop, previously by default, next package was overriding previous one and so on..